system/web/context/ExceptionBean.cfc

```diff
@@ -41,6 +41,20 @@ component accessors="true"{
 		}
 		variables.extraMessage 	= arguments.extraMessage;
 		variables.extraInfo 	= arguments.extraInfo;
+		
+		try {
+			if(structIsEmpty(variables.exceptionStruct) && isObject(arguments.errorStruct)){
```

Comment:
Yes definitely.  However, since we know we are in cf10+ and only ACF exhibits this behavior, let's use duplicate for it then.